File: src/main/java/pro/sky/java/weatherhistoryapp/repository/HistoryRepository.java
package pro.sky.java.weatherhistoryapp.repository;

import org.springframework.data.repository.reactive.ReactiveCrudRepository;
import org.springframework.stereotype.Repository;
import pro.sky.java.weatherhistoryapp.domain.HistoryRecord;

@Repository
public interface HistoryRepository extends ReactiveCrudRepository<HistoryRecord, Integer> {
}

File: src/main/kotlin/pro/sky/java/weatherhistoryapp/repository/HistoryRepository.kt
package pro.sky.java.weatherhistoryapp.repository

import org.springframework.data.repository.kotlin.CoroutineSortingRepository
import org.springframework.stereotype.Repository
import pro.sky.java.weatherhistoryapp.domain.HistoryRecord

@Repository
interface HistoryRepository : CoroutineSortingRepository<HistoryRecord, Int>